build_targets

```diff
@@ -32,3 +32,4 @@ sweet user yes yes
 vayu user yes yes
 violet userdebug yes yes
 zippo userdebug yes no
+haydn userdebug yes no
```

Comment:
add in proper place first

Reply:
also
don't clone stuff via vendor.sh call
https://github.com/crdroidandroid/android_device_xiaomi_haydn/blob/13.0/vendorsetup.sh

Reply:
and have a read again here
https://github.com/crdroidandroid/build_targets#dependencies
currently not proper setup

Reply:
I have removed the vendor.sh call
Added the Haydn at proper place 

Could you please tell me 
What's the problem in dependencies

Regards, Vinayak